AI-and-Analytics/Getting-Started-Samples/IntelPython_daal4py_GettingStarted/README.md

```diff
@@ -1,102 +1,137 @@
-# `Intel® Python daal4py Getting Started` Sample
-This Getting Started sample code show how to do batch linear regression using the python API package daal4py powered by the oneAPI Data Analytics Library (oneDAL). It demonstrates how to use software products that are powered by [oneAPI Data Analytics Library](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onedal.html) and found in the [Intel® oneAPI AI Analytics Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html).
+# Intel® Python daal4py Get Started Sample
 
-| Optimized for                     | Description
+This get started sample code shows how to do batch linear regression using the Python API package daal4py powered by the oneAPI Data Analytics Library (oneDAL). It demonstrates how to use software products that are powered by [oneAPI Data Analytics Library](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onedal.html) and found in the [Intel® oneAPI AI Analytics Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html).
+
+| Property                          | Description
 | :---                              | :---
-| OS                                | 64-bit Linux: Ubuntu 18.04 or higher, 64-bit Windows 10, macOS 10.14 or higher
-| Hardware                          | Intel Atom® Processors; Intel® Core™ Processor Family; Intel® Xeon® Processor Family; Intel® Xeon® Scalable processor family
-| Software                          | Intel® oneAPI AI Analytics Toolkit
-| What you will learn               | basic daal4py programming model for Intel CPU
+| Category                          | Get started sample
+| What you will learn               | Basic daal4py programming model for Intel CPUs
 | Time to complete                  | 5 minutes
 
 ## Purpose
 
-daal4py is a simplified API to Intel® oneDAL that allows for fast usage of the framework suited for Data Scientists or Machine Learning users. Built to help provide an abstraction to Intel® oneDAL for direct usage or integration into one's own framework.
+daal4py is a simplified API to Intel® oneDAL that allows for fast usage of the framework suited for data scientists or machine learning users. Built to help provide an abstraction to Intel® oneDAL for direct usage or integration into one's own framework.
 
 In this sample, you will run a batch Linear Regression model with oneDAL daal4py library memory objects. You will also learn how to train a model and save the information to a file.

```

Comment:
the phrase `oneDAL daal4py library memory objects` doesn't make sense
it probably should be just `daal4py library memory objects`